guides/common/modules/proc_installing-convert2rhel.adoc

```diff
@@ -0,0 +1,27 @@
+[id="installing-convert2rhel_{context}"]
+
+= Installing Convert2RHEL
+
+The following procedure makes Convert2RHEL available to your system.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* *>* *Products* and click *Create Product*.
+. In the *Name* field, enter the name of the product.
+{Project} automatically completes the *Label* field based on your entry for *Name*.
+. Optional: From the *GPG Key* list, select the GPG key for the product.
+. Optional: From the *SSL CA Cert* list, select the SSL CA certificate for the product.
+. You do not need to complete *SSL Client Cert* or *SSL Client Key*.
+. Optional: From the *Sync Plan* list, select an existing sync plan or click *Create Sync Plan* to create a sync plan for your product requirements.
+. In the *Description* field, enter a description of the product.
+. Click *Save*.
+. Click the *Repositories* tab.
+. To add a repository, click *New Repository*.
+. In the *Name* field, enter the name of the repository.
+. From the *Type* list, set the content type to Yum.
+. Do not restrict the architecture or the OS version.
+. Use https://ftp.redhat.com/redhat/convert2rhel/7/os as the upstream URL.
```

Comment:
```suggestion
. Set the *Upstream URL* to `\https://ftp.redhat.com/redhat/convert2rhel/7/os`.
```

Reply:
Oh dear, they just migrated the content to `https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/` and there's self-signed certificate (being work on).

So please modify the URL and add one more step - `Disable the "Verify SSL" checkbox`. We'll hopefully remove this step soon when the certificate issue is resolved.

Reply:
Fixed.